### PR TITLE
DFPL-2241 Court services needed - smoke test enhancement 

### PR DESCRIPTION
**Pull request by Braimah101**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-2241

### Change description ###
As part of the FPL, we need to create a  case to the new Playwright framework. This task involves migrating the test suite and updating any necessary configurations and tests as required.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by chak-shing-lo-justice — DISMISSED** [review]
lgtm

**Review by qasimnawaz-moj — APPROVED** [review]
lgtm

File: playwright-e2e/pages/start-application.ts [hunk 1]
```diff
@@ -153,6 +155,10 @@ export class StartApplication {
     await this.internationalElementsHeading.isVisible();
     await this.internationalElementsHeading.click();
   }
+
+  async courtServicesNeededReqUpdated() {
+    await this.courtServicesNeeded.isVisible();
+    await this.courtServicesNeeded.click();
```
Comment (abigailsmith97) [inline playwright-e2e/pages/start-application.ts:162]: missing '}'

**Review by abigailsmith97 — CHANGES_REQUESTED** [review]

File: playwright-e2e/pages/start-application.ts (hunk 1 above)
Comment (Braimah101) [reply playwright-e2e/pages/start-application.ts:163]: Done

**Review by DanCatchpole — APPROVED** [review]
lgtm!

**Pull request merged**